tests/quantization/test_compressed_tensors.py

```diff
@@ -57,6 +71,11 @@ def use_v0_only(monkeypatch):
 )
 def test_compressed_tensors_w8a8_static_setup(vllm_runner, model_args):
     model_path, strategy, quant_type, shape_0, is_symmetric = model_args
+
```

Comment:
This logic is a bit confusing. What models are and aren't supported by aiter vs Triton?

Reply:
Could you add that as a comment?